sources/packages/web/src/views/institution/student/InstitutionStudentOverawards.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <tab-container :enableCardView="false">
+    <student-overaward-details :studentId="studentId" :showAddedBy="true"
```

Comment:
showAddedBy not required as per https://github.com/bcgov/SIMS/pull/1906/files#r1178278574